Lab#1.ipynb

```diff
@@ -0,0 +1,283 @@
+      "text/plain": [
+       "<Figure size 1440x1080 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "#resize plot\n",
+    "plt.rcParams[\"figure.figsize\"] = (20,15)\n",
+    "fig, ax = plt.subplots(1,1)\n",
+    "plt.tick_params(labelsize = 24)\n",
+    "\n",
+    "d = stats.expon.rvs(loc = 0., scale = 2, size = 100000)\n",
+    "ax.hist(d, 50, density=True)\n",
+    "\n",
+    "x = np.linspace(0.05, 10.05, 2000)\n",
+    "plt.xlim([0.05, 10.05])\n",
+    "ax.plot(x, stats.expon.pdf(x, loc=0, scale=2), 'r-', lw=3, alpha=0.8, label='expon pdf')\n",
+    "\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Suppose we have a radioactive sample (plotted and standardized) that has a measured burst of gamma rays after 4 years (located at x = 4); What is the probability that what we observed at the end of 4 years was produced the background rather than a signal?\n",
```

Comment:
Not sure if this is related to the plot. If not then perhaps split it up and add a small description for it. If so then maybe make it more clear